### DISCUSSION TITLE
Table alias

### DISCUSSION THREAD

**Discussion by averyoldman** [rbatis/rbatis#392] [Q&A] [answered]
Hi all,

I've been trying to find if it's possible to set a table alias, I could only find a thread discussing column alias.
Is anyone willing to share a example on having a table alias?

For example:

`
#[table-alias(ice-cream)]
pub struct iceCream{...}
`
Thanks for any suggestions and help.

Kind regards,

SB

**Comment by averyoldman** [comment, marked as answer]
Found it: crud!(iceCream{}, "ice_cream");

**Reply by zhuxiujia** [reply]
yes,this is
